src/api/transport.js

```diff
@@ -3,5 +3,9 @@ import base64 from 'base-64';
 
 import type { Auth } from './transportTypes';
 
-export const getAuthHeaders = (auth: Auth): { [string]: string } =>
-  auth.apiKey ? { Authorization: `Basic ${base64.encode(`${auth.email}:${auth.apiKey}`)}` } : {};
+export const getAuthHeaders = (auth: Auth): {| Authorization?: string |} =>
+  // The redundant `Authorization` in the `:` case avoids a Flow issue:
+  // https://github.com/facebook/flow/issues/2386
+  auth.apiKey
+    ? { Authorization: `Basic ${base64.encode(`${auth.email}:${auth.apiKey}`)}` }
+    : { Authorization: undefined };
```

Comment:
Hmm, so this actually changes what gets returned -- instead of `{}` we return `{ Authorization: undefined }`. That *probably* doesn't ultimately matter, but that's not immediately obvious -- in at least one call site (in `apiFetch.js`) it propagates through to the calling function's own return value.

How about `Object.freeze({})` instead? I learned about that solution just now from browsing a couple of these issue threads:
https://github.com/facebook/flow/issues/2386#issuecomment-695064325